### PR TITLE
Add workers for eph smoke tests

### PR DESCRIPTION
**Pull request by myersCody**
Fixes [COST-####](https://issues.redhat.com/browse/COST-####)

Changes proposed in this PR:
*
*

Testing Instructions:
1. Run `this command` ...
2. Ingest this data `with this other command` ...
3. Expect to see this result:
   ```
   GET http://localhost:8000/api/endpoint

   {json: response}
   ```
4.
5.

---
Additional Context:

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ DBM_IMAGE=${IMAGE}
 DBM_INVOCATION=$(printf "%02d" $(((RANDOM%100))))
 COMPONENTS="hive-metastore koku presto"  # specific components to deploy (optional, default: all)
 COMPONENTS_W_RESOURCES="hive-metastore koku presto"  # components which should preserve resource settings (optional, default: none)
+SOURCE="appsre"
```
Comment (lcouzens) [inline pr_check.sh:13]: probably want to remove this line

**Review by lcouzens — DISMISSED** [review]
LGTM!

File: koku/masu/processor/ocp/ocp_cloud_updater_base.py [hunk 2]
```diff
@@ -135,9 +135,10 @@ def set_provider_infra_map(self, infra_map):
         """
         for key, infra_tuple in infra_map.items():
             with ProviderDBAccessor(key) as provider_accessor:
-                provider_accessor.set_infrastructure(
-                    infrastructure_provider_uuid=infra_tuple[0], infrastructure_type=infra_tuple[1]
-                )
+                if provider_accessor.provider:
```
Comment (myersCody) [inline koku/masu/processor/ocp/ocp_cloud_updater_base.py:138]: Added to potentially help with: https://issues.redhat.com/browse/COST-2537

**Comment by cgoodfred** [comment]
Should we run full smokes on this just to see how they go? If they go well(ish) and reasonably fast then were probably good, but if they don't go so well, we might want to consider trying to bump resources in ephemeral

**Review by lcouzens — APPROVED** [review]
LGTM

**Pull request merged**